### PR TITLE
RELATED: FET-723 Prevent use of lodash/get

### PR DESCRIPTION
**Pull request by no23reason**
This adds a dep-cruiser rule that disallows the use of lodash/get.
The reasoning is that this function destroys type information (especially
when using the dot notation) and makes refactoring harder. Moreover,
with ?. and ?? operators available, it makes even less sense to use it.

This migrates alle the packages with two exceptions:

* sdk-ui-ext (it has the get key syntax baked in the supported properties logic)
* api-client-bear (there is some key based magic I was too afraid to touch)

I will revisit these in a separate PR and try to limit the use of lodash/get
there as much as possible.

JIRA: FET-723

---

Supported PR commands:

| Command                | Description            |
| ---------------------- | ---------------------- |
| `ok to test`           | Re-run standard checks |
| `extended test`        | BackstopJS tests       |
| `extended check sonar` | SonarQube tests        |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [x] `check` passes
-   [x] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by no23reason** [comment]
extended test

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/877/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3457/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/877/ : UNSTABLE in 19m 09s

**Comment by no23reason** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3456/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/5090/ : SUCCESS in 12m 18s

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/878/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3458/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/878/ : SUCCESS in 19m 29s

File: libs/sdk-ui-pivot/src/impl/data/tests/loadingRenderer.test.tsx [hunk 1]
```diff
@@ -39,33 +39,37 @@ describe("RowLoadingElement", () => {
         expect(wrapper.html()).toEqual('<span class="s-value s-loading-done">3.14</span>');
     });
 
-    describe("'LoadingComponent' color property", async () => {
-        const LoadingRenderer = await createTestRenderer();
+    describe("'LoadingComponent' color property", () => {
```
Comment (lupko) [inline libs/sdk-ui-pivot/src/impl/data/tests/loadingRenderer.test.tsx:42]: out of curiosity... what is wrong with the old version? i did it like this and thought it's ok to do

**Review by lupko — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR1311

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/1027/ : FAILURE in 2m 11s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1297/ : SUCCESS in 21s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/10/ : SUCCESS in 8m 48s (non-voting)
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/5097/ : SUCCESS in 11m 54s

**Pull request merged**